docs/usage.md

```diff
@@ -613,7 +623,7 @@ your project before (via `dotnet build` or `dotnet publish`) and specify `--no-b
 ## casr-libfuzzer
 
 Triage crashes found by libFuzzer based fuzzer
-(C/C++/go-fuzz/Atheris/Jazzer/Jazzer.js/jsfuzz/luzer) or LibAFL based fuzzer
+(C/C++/go-fuzz/Atheris/Jazzer/Jazzer.js/jsfuzz) or LibAFL based fuzzer
```

Comment:
redundant change

Reply:
It's auto change from `./update_usage.py`

Reply:
Can you add luzer to casr-libfuzzer help, please?

Reply:
Done